streamz/core.py

```diff
@@ -1130,7 +1130,14 @@ def make_coro():
         else:
             return gen.with_timeout(timedelta(seconds=timeout), coro)
 
-    if not loop._running:
+    try:
+        loop_is_running = loop._running
+    except AttributeError:
+        try:
+            loop_is_running = loop.asyncio_loop.is_running()
+        except AttributeError:
+            raise NotImplementedError(type(loop))
+    if not loop_is_running:
```

Comment:
I recommend removing this code rather than protecting it.

Reply:
Ok thanks for the pointer. I've done that. The fib_asyncio.py example runs fine now. I tried running the tests but I'm having trouble with my setup on Windows. Travis also seems to be reporting a failing check.